spec/spec_helper.rb

```diff
@@ -1 +1,23 @@
+if ENV['COVERAGE']
+  # In order to archive an expected coverage analysis we need to eager load
+  # all logstash code base, otherwise it will not get a good analysis.
+
+  require "pluginmanager/main"
+  Dir.glob("lib/logstash/patches/*") do |file|
+    require file
+  end
+
+  Dir.glob("lib/logstash/*.rb") do |file|
+    require file
+  end
+
+  Dir.glob("lib/logstash/util/*") do |file|
+    next if ["lib/logstash/util/prctl.rb"].include?(file)
```

Comment:
Actually I don't know, it looks like being a file for MRI and FFI. @jordansissel @ph do you know if this is still necessary? 

```
    if UNAME == "linux"
      require "logstash/util/prctl"
      # prctl PR_SET_NAME allows up to 16 bytes for a process name
      # since MRI 1.9, JRuby, and Rubinius use system threads for this.
      LibC.prctl(PR_SET_NAME, name[0..16], 0, 0, 0)
    end
``` 

^^ is the place where is used in ```lib/logstash/util.rb```.